
For some reason strchr became a macro on my system lately and the protype is
wrong anyway ...

  - Rene Rebe <rene@exactcode.de>

--- pl231src/src/functions.c.vanilla	2005-03-06 12:16:41.160740088 +0100
+++ pl231src/src/functions.c	2005-03-06 12:16:57.089318576 +0100
@@ -636,7 +636,6 @@
 
 	if( hash == 1708 ) { /* $htmldisplay( var1, .. varN ) - convert embedded HTML problem chars (<,>,&) 
 			      * to their html esc sequences for all variables given */
-		char *strchr();
 		for( i = 0; i < nargs; i++ ) {
         		stat = TDH_getvalue( tok, arg[i], TDH_dat, TDH_recid );
 			if( strchr( tok, '&' ) != NULL ) GL_substitute( "&", "&amp;", tok );
